zmon_worker_monitor/builtins/plugins/elasticsearch.py

```diff
@@ -33,24 +33,29 @@ class ElasticsearchFactory(IFunctionFactoryPlugin):
     def __init__(self):
         super(ElasticsearchFactory, self).__init__()
 
+        self._url = None
+
     def configure(self, conf):
         """
         Called after plugin is loaded to pass the [configuration] section in their plugin info file
         :param conf: configuration dictionary
         """
-        return
+        self._url = conf.get('url')
```

Comment:
Ok. Done and tested.

precedence:

+ check command url 
+ entity url
+ config url